src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -18,6 +18,7 @@
     <PublishReadyToRun>true</PublishReadyToRun>
     <!-- Disable crossgen on FreeBSD, NetBSD, illumos and Solaris for now. This can be revisited when we have full support. -->
     <PublishReadyToRun Condition="'$(TargetOS)'=='FreeBSD' Or '$(TargetOS)'=='NetBSD' Or '$(TargetOS)'=='illumos' Or '$(TargetOS)'=='Solaris'">false</PublishReadyToRun>
+    <PublishReadyToRunUseCrossgen2>true</PublishReadyToRunUseCrossgen2>
```

Comment:
so does this use the 5.0 crossgen2 or it will still use the live bits for building?

Reply:
This uses the live bits for building. The ResolveReadyToRunCompilers target below is how the SDK determines which crossgen2 to use.

Reply:
ok thanks.